cms/envs/test.py

```diff
@@ -331,3 +331,7 @@
 
 # API access management -- needed for simple-history to run.
 INSTALLED_APPS += ('openedx.core.djangoapps.api_admin',)
+
+# Used with Email sending
+RETRY_ACTIVATION_EMAIL_MAX_ATTEMPTS = 5
```

Comment:
Because we have to define these variables in tests too. 
I also checked this by setting in common.py but my tests were failing.